runt/src/main.rs

```diff
@@ -141,6 +143,7 @@ async fn run_code(id: String, code: String) -> Result<(), Error> {
         .await?;
 
     // Deserialize the response
+    println!("DEBUG Response: `{}`", response);
```

Comment:
Protip for debug printing -- use `dbg!(response)`. That'll log the line it came from too when you run this.